CONTRIBUTING.md

```diff
@@ -1,99 +1,74 @@
 # Contribution guidelines
+**SimpleSAMLphp welcomes all contributions**. It is impossible to make a product like this without the efforts of many people, so please don't be shy and share your help with us. Even the tiniest contribution can make a difference!
 
-**SimpleSAMLphp welcomes all contributions**. It is impossible to make a product like this without the efforts of many
-people, so please don't be shy and share your help with us. Even the tiniest contribution can make a difference!
-
-This guidelines briefly explain how to contribute to SimpleSAMLphp in an effective manner, making sure to keep high
-quality standards and making it easier for your contributions to make through.
-
-<!-- {{TOC}} -->
```

Comment:
This is not just a comment, but an instruction that some Markdown compilers can use to generate a Table of Contents at this point.